extensions/aris/package.json

```diff
@@ -0,0 +1,98 @@
+{
+	"name": "aris",
```

Comment:
should we use codenames in the product?

Reply:
same comment regarding the directory name.  I think we should use a more official name if possible.

Reply:
We shouldn't have a new extension at all, my understanding was we were going to use the existing big-data-cluster extension here?

Reply:
Yes. I just posted as new extension for easiness of review. This is initial commit, and it will move to big-data-cluster.

Reply:
Generally it would be great to at least let people know that there are major changes like that in the description so they're aware before they start reviewing. And for something like this I know I"d at least prefer you make those kind of changes before sending out for review (unless you have a specific thing you want someone to look at). Right now it's hard to tell what I should spend time looking at since it's unclear exactly how much of this will still be around once you merge it in with the existing extension (for example I assume most of the wrapper stuff is going to go away since the extension should have that stuff set up already)

---
In reply to: [298728868](https://github.com/microsoft/azuredatastudio/pull/6204#discussion_r298728868) [](ancestors = 298728868)

Reply:
fixed.